src/lang/std/engineConnection.ts

```diff
@@ -937,9 +937,14 @@ class EngineConnection extends EventTarget {
               })
               .join('\n')
             if (message.request_id) {
+              const pendingCommand =
+                this.engineCommandManager.pendingCommands[message.request_id]
               console.error(
-                `Error in response to request ${message.request_id}:\n${errorsString}
-    failed`
+                `Error in response to request ${message.request_id}:\n${errorsString}\n\nPending command:\n${JSON.stringify(
+                  pendingCommand,
+                  null,
+                  2
+                )}`
```

Comment:
unrelated, but I had noticed seperately that this error is not very useful if it doesn't show you the request that failed, because the engine doesn't return any info about the request besides it's original id.